tools/test_bundled_libs.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+from sys import argv
+import subprocess
+
+def check_ldd_out(lib, linked_lib, bundled_lib_names, allowed_libs):
+    # Gather all libs that may be linked wiht 'lib' and don't need to be bundled
+    # Entries from 'lib' key should cover all 'lib*' libs
```

Comment:
This comment is outdated, now it is `""` key.
```suggestion
    # Entries from '' (empty string) key are allowed for any lib
```

Reply:
Will do

Reply:
On second thought this comment is about wildcards - that if you use "lib" as a key int the dict then all libraries "lib.*" will be covered by that. So comment is not outdated.